Sources/FluentKit/Properties/ID.swift

```diff
@@ -56,6 +56,12 @@ public final class IDProperty<Model, Value>
         }
     }
 
+    /// Initializes an `ID` property with the key `.id` and type `UUID`.
+    ///
+    /// If the property's type is not `UUID` or the key is not `.id`, the initializer will
+    /// fatal error. This is so Fluent can support databases like MonogoDB more natively.
```

Comment:
Nit - tighten up the language
```suggestion
    /// fatal error. This is so Fluent can natively support databases like MonogoDB.
```

Reply:
Double nit (and spelling!):
```suggestion
    /// fatal error. This allows Fluent to natively support databases like MongoDB.
```